texts/pride_prejudice.txt

```diff
@@ -0,0 +1,1833 @@
+;	"single" n sg
+"<?>"
+	"?" sent
+”
+"<.>"
+	"." sent
+
+
+      “
+"<Oh>"
+	"oh" ij
+"<!>"
+	"!" sent
+"<Single>"
+	"single" adj
+;	"single" n sg
+"<,>"
+	"," cm
+"<my>"
+;	"my" det pos sp
+	"prpers" det pos p1 mf sg
+"<dear>"
+;	"dear" adv
+	"dear" adj sint
+"<,>"
+	"," cm
+"<to>"
+	"to" pr
+"<be>"
+	"be" vbser inf
+"<sure>"
+;	"sure" adv
+	"sure" adj sint
+"<!>"
+	"!" sent
+"<A>"
+	"a" det ind sg
+"<single>"
+	"single" adj
+;	"single" n sg
+"<man>"
+	"man" n sg
+"<of>"
+	"of" pr
+"<large>"
+	"large" adj sint
+"<fortune>"
+	"fortune" n sg
+"<;>"
+	";" sent
+"<four>"
+	"four" num pl
+"<or>"
+	"or" cnjcoo
+"<five thousand>"
+	"five thousand" num pl
+"<a>"
+	"a" det ind sg
+"<year>"
+	"year" n sg
+"<.>"
+	"." sent
+"<What>"
+	"what" det itg sp
+;	"what" prn itg mf sp
+;	"what" prn rel nn mf sg
+"<a>"
+	"a" det ind sg
+"<fine>"
+;	"fine" adv
+;	"fine" n sg
+	"fine" adj sint
+;	"fine" vblex inf
+;	"fine" vblex pres
+;	"fine" vblex imp
+"<thing>"
+	"thing" n sg
+"<for>"
+;	"for" cnjadv
+	"for" pr
+"<our>"
+;	"our" det pos sp
+	"prpers" det pos p1 mf pl
+"<girls>"
+	"girl" n pl
+"<!>"
+	"!" sent
+”
+"<.>"
+	"." sent
+
+
+      “
+"<How>"
+;	"how" adv
+;	"how" preadv
+	"how" adv itg
+"<so>"
+	"so" adv
+;	"so" preadv
+;	"so" cnjadv
+"<?>"
+	"?" sent
+"<How>"
+;	"how" adv
+;	"how" preadv
+	"how" adv itg
+"<can>"
+;	"can" n sg
+;	"can" vbmod inf
+	"can" vbmod pres
+;	"can" vblex inf
+;	"can" vblex pres
+;	"can" vblex imp
+"<it>"
+	"prpers" prn subj p3 nt sg
+;	"prpers" prn obj p3 nt sg
+"<affect>"
+	"affect" vblex inf
+;	"affect" vblex pres
+;	"affect" vblex imp
+"<them>"
+;	"that" prn dem pl
+	"prpers" prn obj p3 mf pl
+"<?>"
+	"?" sent
+”
+"<.>"
+	"." sent
+
+
+      “
+"<My>"
+;	"my" det pos sp
+	"prpers" det pos p1 mf sg
+"<dear>"
+;	"dear" adv
+	"dear" adj sint
+"<Mr.>"
+	"Mr." n sg
+"<Bennet>"
+	"Bennet" np cog sg
+;	"Bennet" np ant f sg
+"<,>"
+	"," cm
+” 
+"<replied>"
+;	"reply" vblex pp
+	"reply" vblex past
+"<his>"
+;	"his" adj pos pred
+;	"his" det pos sp
+;	"prpers" prn pos p3 m sg
+	"prpers" det pos p3 m sg
+"<wife>"
+	"wife" n sg
+"<,>"
+	"," cm
+ “
+"<how>"
+;	"how" adv
+;	"how" preadv
+	"how" adv itg
+"<can>"
+;	"can" n sg
+;	"can" vbmod inf
+	"can" vbmod pres
+;	"can" vblex inf
+;	"can" vblex pres
+;	"can" vblex imp
+"<you>"
+;	"you" prn pers p2 mf sg
+	"prpers" prn subj p2 mf sp
+;	"prpers" prn obj p2 mf sp
+"<be>"
+	"be" vbser inf
+"<so>"
+;	"so" adv
+	"so" preadv
+;	"so" cnjadv
+"<tiresome>"
+	"tiresome" adj
+"<!>"
+	"!" sent
+"<You>"
+;	"You" np cog sg
+;	"you" prn pers p2 mf sg
+	"prpers" prn subj p2 mf sp
+;	"prpers" prn obj p2 mf sp
+"<must>"
+;	"must" n sg
+;	"must" vbmod inf
+	"must" vbmod pres
+"<know>"
+	"know" vblex inf
+;	"know" vblex pres
+;	"know" vblex imp
+"<that>"
+	"that" cnjsub
+;	"that" det dem sg
+;	"that" prn dem mf sg
+;	"that" prn rel an mf sp
+"<I>"
+	"prpers" prn subj p1 mf sg
+"<am>"
+	"be" vbser pres p1 sg
+"<thinking>"
+;	"thinking" n sg
+;	"think" vblex pprs
+;	"think" vblex subs
+	"think" vblex ger
```

Comment:
pretty sure this is a participle

Reply:
Present participle it is, then. I thought it was gerund since, as Spanish speaker I am, the Spanish equivalent would be a gerund.